src/main/java/pl/futurecollars/invoicing/db/file/FilesService.java

```diff
@@ -6,7 +6,9 @@
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
 import java.util.List;
+import org.springframework.stereotype.Service;
 
+@Service
```

Comment:
Skoro tworzysz beana fileService w DbConfiguration to ta adnotacja nie jest Ci potrzebna.